examples/jsm/nodes/materials/NodeMaterial.js

```diff
@@ -232,6 +233,12 @@ class NodeMaterial extends ShaderMaterial {
 
 		}
 
+		if( this.envMapIntensityNode && node ) {
```

Comment:
Sure, I will rename it to `envIntensityNode`, which is also more consistent with `envNode` 👍

Reply:
> Sure, I will rename it to envIntensityNode, which is also more consistent with envNode 👍

It makes sense for `WebGLRenderer`, but I don't think that's the case here, because this block is already aligned with `envNode`. I think this reinforces that we should remove this support for `WebGPURenderer` and leave it only in `WebGLRenderer`.

For `WebGPURenderer` this should be used as `.mul()`, no other `*Intensity` property suffix is required for the `NodeMaterial`.

If you can remove the modifications in `StandardMaterialEditor` and `NodeMaterial` I think there would be no other impediment.

Reply:
Ok, makes sense!
I removed the support for `WebGPURenderer` in `NodeMaterial` and added a comment in the code for clarification:

```
// Only used by WebGLRenderer for scaling the environment illumination
// For WebGPURenderer, scaling of the environment intensity can be achieved as envNode.mul(...)
this.envMapIntensityNode = null;
```

I also removed the changes in `StandardMaterialEditor`.

I understood you, that we would then keep its name as `envMapIntensityNode` because it is only for WebGL support and it would be consistent with the already existing `envMapIntensity` property of a material.